<?xml version="1.0"?>
<launch>
  <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="ev3_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <param name="autorepeat_rate" value="10.0" />
  </node>

  <!-- teleop node -->
  <node pkg="ev3_teleop" type="ev3_teleop_joy" name="ev3_teleop" output="screen">
    <param name="axis_linear" value="3" type="int"/>
    <param name="axis_angular" value="0" type="int"/>
    <param name="axis_deadman" value="4" type="int"/>
    <param name="scale_linear" value="1" type="double"/>
    <param name="scale_angular" value="1" type="double"/>
  </node>
</launch>
